src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectTodo: Todo,
+  setSelectTodo: (todo: Todo | null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectTodo,
+  setSelectTodo,
+}) => {
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(selectTodo.userId)
```

Comment:
The useEffect is missing a dependency on 'selectTodo.userId'. Add it to the dependency array